api/overdrive.py

```diff
@@ -1364,10 +1363,6 @@ def should_stop(self, start, api_description, is_changed):
             self.log.error("Got invalid date: %s", date_added)
             return False
 
-        # The time stored in the database is UTC, but it's stored
-        # without any time zone information. Add that information so
-        # we can compare it against the date we got from Overdrive.
-        start = pytz.utc.localize(start)
```

Comment:
Do we know that in all cases this has timezone info attached? If not maybe we should use this helper that handles the case without timezone and the case with timezone: 
https://github.com/ThePalaceProject/circulation/blob/main/core/util/datetime_helpers.py#L35-L49

On the other hand this is much more clear if we know we always have a timezone here.

Reply:
Good question. I verified that `start` originates in a `timestamp with time zone` field in the `timestamps` table, so should be either `None` or a `datetime` with a time zone.